docs/docker-integration/cloudformation.rst

```diff
@@ -43,7 +43,7 @@ Once it's up and running we'll guide you through a tutorial to deploy a sample a
 		</div>
 		<div class="step-stages__step">
 			<img src="../_images/02-create-key.png" alt="Creating a new AWS key pair"/>
-            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>flocker-test</strong>. You'll need this later, so <strong>make a note of it</strong>.</span>
+            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>username-flocker-test</strong>. You'll need this later, so <strong>make a note of it</strong>.</span>
```

Comment:
Maybe we should put "username" in italics, to indicate that this is a "meta variable" and not the literal word "username"?